app/agents_v2/primary_agent/reasoning/safety_redactor.py

```
@@ -0,0 +1,301 @@
+"""
+Safety Redactor for reasoning UI content.
+
+Ensures no sensitive information, system prompts, or internal details
+leak through the reasoning UI shown to users.
+"""
+
+import re
+import logging
+from typing import Dict, List, Any, Optional
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)